src/widgets-service.js

```diff
@@ -1,4 +1,5 @@
 var _ = require('underscore');
+var URI = require('urijs');
```

Comment:
dashboard should not know about urls anyways, it's a embed/html responsability (maybe using a helper). Anyways, I would use the simplest solution to avoid npm module bloating (and complains about javascript file size :blink: :blink:)